aws-spi-pekko-http/src/test/java/org/apache/pekko/stream/connectors/awsspi/s3/S3Test.java

```diff
@@ -44,7 +43,7 @@
 
 import static org.junit.Assert.assertEquals;
 
-public class S3Test extends JUnitSuite {
+public class S3Test {
```

Comment:
AFAICT, using `JUnitSuite` from scalatest+junit is not needed, and we can remove dependency on `"org.scalatestplus" %% "junit-4-13"`